### PR TITLE
Add recording link for 04/20 community call

### PR DESCRIPTION
**Pull request by AaronCrawfis**

### CODE REVIEW

**Review by jasonviviano — APPROVED** [review]

**Review by Reshrahim — APPROVED** [review]

File: docs/content/community.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ To present a specific topic on the next Radius community call start by filling o
...
-| Thursday Apr 20th 9:00am Pacific Time (PST) | v0.19 release | [Teams](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MzNmOTQ1YTctOTVjNi00ZjdhLTk1YjgtMzExZWYwNmRlMjU1%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22a6a9b210-8672-47c2-b58f-0aa092ed4b39%22%7d) | TBD |
+| Thursday Apr 20th 9:00am Pacific Time (PST) | v0.19 release | [Teams](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MzNmOTQ1YTctOTVjNi00ZjdhLTk1YjgtMzExZWYwNmRlMjU1%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22a6a9b210-8672-47c2-b58f-0aa092ed4b39%22%7d) | [Stream](https://microsoft.sharepoint.com/:v:/r/teams/radius/Shared%20Documents/General/Community%20Calls/2023-04-20%20Project%20Radius%20Community%20Call.mp4?csf=1&web=1&e=8seDme) |
```
Comment (willtsai) [inline docs/content/community.md:34]: Interestingly, it says I don't have access, though I'm pretty sure I'm a member of the Teams space for Radius. Is this just an issue on my end or would the community run into this as well?

Comment (jasonviviano) [reply docs/content/community.md:34]: For extra context, I was able to access it right now.

Comment (AaronCrawfis) [reply docs/content/community.md:34]: @willtsai are you using your work browser to try and access? This link should only work for preview members who are signed in with the account they use for public release access (which for us is our Microsoft account). If you are using your work browser and getting this error, could you paste a screenshot of the error you're getting?

Comment (willtsai) [reply docs/content/community.md:34]: @AaronCrawfis I am indeed using my work browser authenticated into my Microsoft account to try access but running into the permissions issue. I've even tried on a private window, logged into my Microsoft account, and still see the issue. Here's a screenshot:

image

Comment (AaronCrawfis) [reply docs/content/community.md:34]: Welp, looks like you weren't added to our Teams team 😅. Added:
image

Comment (willtsai) [reply docs/content/community.md:34]: Seems like it was an access-related issue, resolved after @AaronCrawfis added me

**Pull request merged**